src/_layouts/page.html

```diff
@@ -90,41 +90,29 @@
     <!-- End Google Tag Manager (noscript) -->
 
     <header>
-      <nav class="dark-bg">
-        <input type="checkbox" />
-        <label>&equiv;</label>
-        <a class="home-button" href="/" aria-label="BeakBox Home page">
-          <img class="clickable" alt="Logo" src="/imgs/title.svg" />
+      <nav class="main-nav">
```

Comment:
I've decided to keep the class here but renamed it to "main-nav" - to keep using BEM and differentiate it from the nav element in the blog section (I might see if we can improve it in the next pr)